example/README.md

```diff
@@ -114,16 +124,17 @@ mkShell {
 }
 ```
 
-As soon as you save, `lorri` starts building the changes.
+Then tell the daemon to build the changes:
 
+```bash
+$ lorri _ping $(readlink -f shell.nix)
+```
```

Comment:
This is the point where we want to show the user that lorri automatically picks up changes. Since that only happens with `direnv` integration anymore, we should move the sentence

> As soon as you save, `lorri` starts building the changes.

down into the direnv part.